src/virtio/include/config/virtio_pci.h

```diff
@@ -0,0 +1,92 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
I don't think we should include PCI or any other configs until we have them implemented.